rgd/geodata/tests/test_demo_data.py

```diff
@@ -0,0 +1,25 @@
+from io import StringIO
+import pytest
+
+from django.core.management import call_command
+
+from rgd.geodata.management.commands.demo_data import SUCCESS_MSG
+
+
+def _call_command(*args, **kwargs):
+    out = StringIO()
+    call_command(
+        "demo_data",
+        *args,
+        stdout=out,
+        stderr=StringIO(),
+        **kwargs,
+    )
+    return out.getvalue()
+
+
+@pytest.mark.skip
```

Comment:
Marking test to skip because it takes ~107.25s with the current list of data (I plan to add more)

I'm going to keep the test here for manual running (that is, comment this line locally to make sure everything is working). It's just too much to put on the CIs and all other normal dev testing